### ISSUE TITLE
VK_EXT_fragment_density_map mentions non-existent flag: VK_IMAGE_CREATE_FRAGMENT_DENSITY_MAP_OFFSETS_BIT_EXT

### ISSUE THREAD

**Issue by wheeland** [KhronosGroup/Vulkan-Docs#1742]
https://github.com/KhronosGroup/Vulkan-Docs/blob/main/appendices/VK_EXT_fragment_density_map.txt mentions on line 146 that `// All attachments must be created with VK_IMAGE_CREATE_FRAGMENT_DENSITY_MAP_OFFSETS_BIT_EXT`. 

However, this flag doesn't exist in the Vulkan headers nor in the rest of the Vulkan docs and is never mentioned anywhere else.

**Comment by wheeland** [comment]
There is `VK_IMAGE_CREATE_FRAGMENT_DENSITY_MAP_OFFSET_BIT_QCOM` from `VK_QCOM_fragment_density_map_offset`, but this extension is not required by `VK_EXT_fragment_density_map`, so the comment doesn't really seem to make a lot of sense here.

**Comment by jackohound** [comment]
Agree, the comment is wrong as it stands and this code sample is not intended to be dependent on VK_QCOM_fragment_density_map_offset extension.  I think the line should simply be removed.
I'll submit a fix.

**Comment by mnetsch** [comment]
Agreed, I think I made a copy+paste error when writing the offsets extension

**Comment by speedym** [comment]
@jackohound Note that there is the following wording in the spec: ```Subpass color and depth attachments can be created as subsampled, which can help to further optimize rendering in areas of low density.``` but the comment in the code says: ```Color attachments can be created with VK_IMAGE_CREATE_SUBSAMPLED_BIT_EXT```.

Maybe the code comment could be clarified to include both depth & color attachments?

**Comment by oddhack** [comment]
This was fixed a few months ago but we neglected to close it at the time.
